dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataaccess/FileAccessIO.java

```diff
@@ -574,7 +578,7 @@ private String getDatasetDirectory() throws IOException {
 
             baseName = this.getDataFile().getStorageIdentifier();
 
-            datasetDirectoryPath = this.getDataFile().getOwner().getFileSystemDirectory();
+            datasetDirectoryPath = this.filesRootDirectory;
```

Comment:
This is replaced incorrectly it should be rather `this.getDataFile().getOwner().getFileSystemDirectory(filesRootDirectory)`